MEAnalysis/crab_vhbb/multicrab.py

```diff
@@ -42,26 +42,43 @@
     "/DoubleEG/Run2016B-PromptReco-v2/MINIAOD",
     "/DoubleEG/Run2016C-PromptReco-v2/MINIAOD",
     "/DoubleEG/Run2016D-PromptReco-v2/MINIAOD",
+    "/DoubleEG/Run2016E-PromptReco-v2/MINIAOD",
+    "/DoubleEG/Run2016F-PromptReco-v2/MINIAOD",
 
     "/DoubleMuon/Run2016B-PromptReco-v1/MINIAOD",
     "/DoubleMuon/Run2016B-PromptReco-v2/MINIAOD",
     "/DoubleMuon/Run2016C-PromptReco-v2/MINIAOD",
     "/DoubleMuon/Run2016D-PromptReco-v2/MINIAOD",
+    "/DoubleMuon/Run2016E-PromptReco-v2/MINIAOD",
+    "/DoubleMuon/Run2016F-PromptReco-v2/MINIAOD",
 
     "/MuonEG/Run2016B-PromptReco-v1/MINIAOD",
     "/MuonEG/Run2016B-PromptReco-v2/MINIAOD",
     "/MuonEG/Run2016C-PromptReco-v2/MINIAOD",
     "/MuonEG/Run2016D-PromptReco-v2/MINIAOD",
+    "/MuonEG/Run2016E-PromptReco-v2/MINIAOD",
+    "/MuonEG/Run2016F-PromptReco-v2/MINIAOD",
 
     "/SingleElectron/Run2016B-PromptReco-v1/MINIAOD",
     "/SingleElectron/Run2016B-PromptReco-v2/MINIAOD",
     "/SingleElectron/Run2016C-PromptReco-v2/MINIAOD",
     "/SingleElectron/Run2016D-PromptReco-v2/MINIAOD",
+    "/SingleElectron/Run2016E-PromptReco-v2/MINIAOD",
+    "/SingleElectron/Run2016F-PromptReco-v2/MINIAOD",
 
     "/SingleMuon/Run2016B-PromptReco-v1/MINIAOD",
     "/SingleMuon/Run2016B-PromptReco-v2/MINIAOD",
     "/SingleMuon/Run2016C-PromptReco-v2/MINIAOD",
     "/SingleMuon/Run2016D-PromptReco-v2/MINIAOD",
+    "/SingleMuon/Run2016E-PromptReco-v2/MINIAOD",
+    "/SingleMuon/Run2016F-PromptReco-v2/MINIAOD",
+
+    "/BTagCSV/Run2016B-PromptReco-v1/MINIAOD",
```

Comment:
You're right! I was pretty user that the sixjet triggers were the the BTagCSV, probably it was a mistake from the TSG.
Anyway, I'm adding the JetHT!